assets/models/system/snowflake-snowflake-arctic-base/spec.yaml

```diff
@@ -0,0 +1,26 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: snowflake-snowflake-arctic-base
```

Comment:
why are these folder names and model names have double "snowflake"?
should be snowflake-arctic-base

Reply:
Chnaged the foldername and model name